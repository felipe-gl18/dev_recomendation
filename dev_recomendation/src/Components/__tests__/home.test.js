import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../homeComponent';

it('snapshot home', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
})