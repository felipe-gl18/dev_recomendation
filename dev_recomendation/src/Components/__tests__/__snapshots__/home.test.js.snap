// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot home 1`] = `
<main>
  <div
    className="apresentation"
  >
    <div
      className="apresentationText"
    >
      <h1>
        <strong>
          DEV
        </strong>
        <br />
        RECOMENDATION
      </h1>
      <p>
        HERE YOU CAN FIND OUT WHERE AND HOW TO 
        <br />
         ACCESS THE BEST CONTENTS FOR 
        <br />
         DEVOLOPERS
      </p>
    </div>
  </div>
  <form
    action="/"
    method="GET"
  >
    <input
      id="search"
      name="search"
      placeholder="Search for some recomendation"
      type="text"
    />
    <button>
      <i
        className="fas fa-search"
      />
    </button>
  </form>
  <div
    className="recomendations"
  >
    <div
      className="recomendations-item"
    >
      <div
        className="recomendations-item-image"
      >
        <img
          alt="illustration"
          src="/imgs/main-photo.jpg"
        />
      </div>
      <div
        className="recomendations-item-content"
      >
        <p
          className="recomendations-item-title"
        >
          React advenced course
        </p>
        <p
          className="recomendations-item-description"
        >
          A react course with advanced content
        </p>
        <button>
          <a>
            See more
          </a>
        </button>
      </div>
    </div>
  </div>
</main>
`;
